addons/sales_team/security/ir.model.access.csv

```diff
@@ -2,3 +2,6 @@ id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
 access_crm_team,crm.team,model_crm_team,base.group_user,1,0,0,0
 access_crm_team_user,crm.team.user,model_crm_team,sales_team.group_sale_salesman,1,0,0,0
 access_crm_team_manager,crm.team.manager,model_crm_team,sales_team.group_sale_manager,1,1,1,1
+access_crm_tag,crm_tag,model_crm_tag,base.group_user,1,0,0,0
```

Comment:
While we are at it, could you make a small separate commit that resets rights on crm tag to 0, 0, 0, 0 for users ? They have no need to see them.

Reply:
ok